WordPress/Classes/Services/ReaderTopicService.h

```diff
@@ -238,6 +238,16 @@ extern NSString * const ReaderTopicFreshlyPressedPathCommponent;
  */
 - (ReaderSiteTopic *)findSiteTopicWithFeedID:(NSNumber *)feedID;
 
+/**
+ Find a site topic by siteID.
+ Combines `findSiteTopicWithSiteID` and `findSiteTopicWithFeedID` to get a topic by ID.
+ Useful when it's not known if the siteID is internal or external.
+ 
+ @param siteID The site id of the topic
+ @return A matched site topic
+ */
+- (ReaderSiteTopic *)siteTopicForSiteID:(NSNumber *)siteID;
```

Comment:
Being so similar to `findSiteTopicWithSiteID` I wonder if we could call it something like `siteTopicForSiteOrFeedID` to better distinguish it and make usage more obvious at a glance? Or even something like `siteTopicForUnknownID` or `siteTopicForGenericID`.